applications/standalone/src/mock/context.ts

```diff
@@ -57,18 +59,55 @@ const getValueStats = <S = unknown>(value: unknown): S => {
   return undefined as S
 }
 
-const getFilteredValue = <T = unknown>(value: T, filter?: TabularDataFilter): T => {
+const sortTable = (table: Table, sortingMethod?: DataSortingMethod): Table => {
+  const sortingColumn = sortingMethod?.column
+  const sortingDirection = sortingMethod?.direction
+
+  if (sortingColumn == null) return table
+  return arrowUtils.sortTable(table, (rowA, rowB) => {
+    if (sortingDirection == null || sortingDirection == 'default') return 0
+
+    if (rowA[sortingColumn] < rowB[sortingColumn]) return sortingDirection == 'asc' ? -1 : 1
+    if (rowA[sortingColumn] > rowB[sortingColumn]) return sortingDirection == 'asc' ? 1 : -1
+    return 0
+  })
+}
+
+const filterTable = (table: Table, condition?: DataFilterCondtion): Table => {
+  // NOTE: in mock implementation only doing 'contains' filtering
+  let filteredTable = table
+  for (const item of condition?.items ?? []) {
+    if (item.operator !== 'contains') continue
+    if (item.value == null || item.value == '') continue
+
+    filteredTable = arrowUtils.filterTable(filteredTable, row => {
+      return String(row[item.column]).includes(String(item.value))
```

Comment:
We could make this case-insensitive for more user-friendliness

Reply:
That's probably only developer-friendliness. This function is used in the mock backend seen only by us. It's just there to check that filtering works in principle. But I updated it anyway.